### ISSUE TITLE
Error docker elasticsearch image on Apple Silicon chip

### ISSUE THREAD

**Issue by wandersoncferreira** [threatgrid/ctia#1135]
Executing `docker-compose ... up` for dev environment produces an error on Apple Silicon chip with the Elasticsearch image:

```
ERROR: no matching manifest for linux/arm64/v8 in the manifest list entries
```

The Docker Apple Silicon states that [not all images are available for arm64](https://docs.docker.com/docker-for-mac/apple-silicon/#known-issues) .

A possible solution is to add `platform: linux/x86_64`[here](https://github.com/threatgrid/ctia/blob/e92263ff7a021335c6a4990602f6b12159aa566e/containers/dev/docker-compose.yml#L7).

**Comment by ereteog** [comment]
FYI @turbodog99 [investigated it in IROH](https://github.com/advthreat/iroh/pull/4883).
In CTIA there is one more constraint, we need ES5 and ES7 until the migration (there are breaking changes).
If the solution you propose works here it's perfect.

**Comment by wandersoncferreira** [comment]
Thanks for pointing that out @ereteog , unfortunately the solution proposed only enabled the docker compose to proceed and spin the images, however, the ES5 breaks when docker try to start it. Then, this should not be considered a solution to this issue.

Definitely a good fix is to create a new Dockerfile targeting M1 for ES5. (ES7 already has support for aarm64)

**Comment by agzam** [comment]
@wandersoncferreira Since not all of us on M1 Macs (I'm not), I think you'd have to test it out and ask @turbodog99 to check as well

**Comment by DeLaGuardo** [comment]
Please check this [PR](https://github.com/threatgrid/ctia/pull/1288). Some changes allow to build ES5 docker image locally and using it on M1 Macs.

TLDR;

Run once — `docker compose -f containers/dev/docker-compose.yml -f containers/dev/m1-docker-compose.yml build`
Then just use the command `docker compose -f containers/dev/docker-compose.yml up` to spin up the cluster.

**Comment by gbuisson** [comment]
I think we can close this issue now, let me know if you still have issues, thanks @DeLaGuardo !
